presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -428,6 +450,14 @@ private void initializePositions(int batchSize)
         }
     }
 
+    private int applyFilterFunctionsWithNoInputs(int[] positions, int positionCount)
```

Comment:
applyFilterFunctionsWithNoInputs -> applyFilterFunctionWithNoInputs (plural to singular)